core/new-gui/src/app/workspace/component/product-tour/product-tour.component.html

```diff
@@ -1,3 +1,10 @@
-<p>
-  product-tour works!
-</p>
+<tour-step-template>
+  <ng-template let-step="step">
+    <p class="tour-step-content" [innerHTML] = 'tourService.currentStep.content'></p>
+    <div class="tour-step-navigation">
+      <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-light" (click)="tourService.prev()">« Prev</button>
```

Comment:
I can't see the Prev/Next buttons in my browser, the CSS seems also broken, please fix that.

Reply:
Prev/Next buttons will appear only if we have multiple steps